src/rgw/const.py

```diff
@@ -21,6 +21,8 @@
 SERVICE_NAME = f'{COMPONENT_NAME}_setup' # rgw_setup
 INSTALL_PATH = '/opt/seagate/cortx'
 RGW_INSTALL_PATH = f'{INSTALL_PATH}/{COMPONENT_NAME}'
+TIMEOUT = 60
+CONSUL_LOCK_KEY = f'component>{COMPONENT_NAME}>volatile>{COMPONENT_NAME}_lock'
```

Comment:
Please add actual lock key as code comment so that these keys can be easily reflected in grep/find commands.
e.g. CONSUL_LOCK_KEY = f'component>{COMPONENT_NAME}>volatile>{COMPONENT_NAME}_lock' **# rgw_lock**